frontend/src/components/ModalContent.vue

```diff
@@ -125,59 +122,83 @@ a {
   text-transform: none;
 }
 
-.buttons-list {
-  margin: 0.5rem 2rem;
+h1 {
+  font-family: $font-header;
+  font-size: 2.5rem;
+  color: $white;
+  word-break: break-word;
+  margin-right: auto;
+  @media only screen and (max-width: $phone) {
+    font-size: 2rem;
+    text-align: center;
+    margin-right: 0;
+  }
 }
 
-.text-list {
-  margin: 0.25rem 3rem;
+h2 {
+  font-family: $font-content;
+  font-size: 1.5rem;
+  color: black;
+  font-weight: bold;
+  word-break: break-word;
+  @media only screen and (max-width: $phone) {
+    text-align: center;
+  }
 }
 
-.buttons {
-  margin: 1rem 0.5rem 1rem 0;
+.buttons-close {
+  background-color: $blue !important;
```

Comment:
is `!important` needed? I guess you can add another class to the element and then you don't have to use such an overwriting

Reply:
Yes, this is the kind of moment when not using important results in nothing. So either I can use in-line prop where I cannot use $var and need to repeat $blue var's hex or I can use important to overrride whatever vuetify understands. I didn't figure out how to pass global $blue into html part of code

Reply:
**Final changes:**
I dropped v-btn altogether so that's not an issue no more